ansible/roles/touchforms/templates/localsettings.py.j2

```diff
@@ -3,7 +3,8 @@
 #
 import os
 
-URL_ROOT = "https://{{ SITE_HOST }}/a/{{ '{{' }}DOMAIN{{ '}}' }}"
+URL_HOST = "https://{{ SITE_HOST }}"
+URL_ROOT = "%s/a/{{ '{{' }}DOMAIN{{ '}}' }}" % URL_HOST
```

Comment:
This is trying to produce the following `/a/{{ DOMAIN }}` which then gets rendered by touchforms in a horribly complicated way: https://github.com/dimagi/touchforms/blob/f49f796ddf21ca68f4e0bce5df97d2229a37d56f/touchforms/backend/util.py#L124-L124

The weird syntax is necessary to escape the braces however it would be much clearer if we use the `raw` template tag:
```
{% raw %}
URL_ROOT = "%s/a/{{ DOMAIN }}" % URL_HOST
{% endraw %}
```

Alternately we could just change the syntax to something that doesn't need to be escaped.